tests/acceptance/features/bootstrap/DataExporterContext.php

```diff
@@ -256,14 +280,52 @@ private function assertFilePhysicallyExistInLastExport($filename, $message = '')
 	/**
 	 * @param string $filename
 	 * @param string $content
+	 * @param bool $trash
 	 *
 	 * @return void
 	 */
-	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content) {
-		$this->featureContext->theFileWithContentShouldExistInTheServerRoot(
-			self::path("$this->lastExportPath/files/$filename"),
-			$content
-		);
+	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content, $trash = false) {
+		if ($trash) {
+			$this->featureContext->listTrashbinFileInServerRoot(self::path("$this->lastExportPath/files_trashbin"));
+			$matches = [];
+			foreach (HttpRequestHelper::getResponseXml($this->featureContext->getResponse())->data->element as $item) {
+				$file = (string)$item;
+				if (\strpos($file, $filename) === 0) {
+					$matches[] = $file;
+				}
+			}
+			$fileFound = false;
+			$fileCount = \count($matches);
+			foreach ($matches as $match) {
+				$path = "$this->lastExportPath/files_trashbin/$match";
+				$this->featureContext->readFileInServerRoot($path);
+				Assert::assertSame(
+					200,
+					$this->featureContext->getResponse()->getStatusCode(),
+					"Failed to read the file {$path}"
+				);
+				$fileContent = HttpRequestHelper::getResponseXml($this->featureContext->getResponse());
+				$fileContent = (string)$fileContent->data->element->contentUrlEncoded;
+				$fileContent = \urldecode($fileContent);
+
+				while ($fileCount > 0) {
```

Comment:
$fileCount will be already zero in the next loop

Reply:
Yeah that's right. For single file $fileileCount will be zero here so the loop won't execute next time but if there are multiple files then the loop must execute more than once until it becomes zero.